src/main/java/org/pac4j/vertx/cas/VertxSharedDataLogoutHandler.java

```diff
@@ -71,9 +71,7 @@ public void destroySession(WebContext context) {
         final String sessionId = getSessionId(ticket);
```

Comment:
Those were added for the initial release. If we can make it easier, so much the better, though you will still need some vertx specific stuff for the actual removal/addition into whatever persistence we use (vert.x shared data should be our default).